modules/govuk_jenkins/templates/jobs/ask_export.yaml.erb

```
@@ -0,0 +1,81 @@
+---
+- scm:
+    name: govuk-ask-export
+    scm:
+      - git:
+          url: git@github.com:alphagov/govuk-ask-export.git
+          branches:
+            - master
+
+- job:
+    name: govuk-ask-export
+    display-name: GOV.UK Ask Export
+    project-type: freestyle
+    <% prod_description = "This job exports responses from the Ask service so that they can be used by the relevant departments" %>
+    <% integration_description = "This is for testing so doesn't run unless you tell it to" %>
+    description: <%= @run_daily ? prod_description : integration_description %>
+    scm:
+      - govuk-ask-export
+    logrotate:
+        numToKeep: 100
+    <% if @run_daily %>
+      triggers:
+        - timed: |
+            TZ=Europe/London
+            0 1 * * *
```

```suggestion
    triggers:
      - timed: |
          TZ=Europe/London
          0 1 * * *
```

The nesting in this conditional will break the yaml as it stands.

---

Thanks, gone!